src/main/java/org/datadog/jenkins/plugins/datadog/publishers/DatadogComputerPublisher.java

```diff
@@ -57,12 +56,12 @@ public long getRecurrencePeriod() {
     }
 
     @Override
-    protected void doRun() throws Exception {
+    public void doRun() throws Exception {
```

Comment:
```suggestion
    protected void doRun() throws Exception {
```
Move your tests files to the right package and you should not have to update visibility of the method.